tslearn/datasets.py

```diff
@@ -214,20 +250,19 @@ def load_dataset(self, dataset_name):
         None
         """
         full_path = os.path.join(self._data_dir, dataset_name)
-        fname_train = self._filenames.get(dataset_name, dataset_name) + "_TRAIN.txt"
-        fname_test = self._filenames.get(dataset_name, dataset_name) + "_TEST.txt"
+        fname_train = dataset_name + "_TRAIN.txt"
+        fname_test = dataset_name + "_TEST.txt"
         if not os.path.exists(os.path.join(full_path, fname_train)) or \
             not os.path.exists(os.path.join(full_path, fname_test)):
             url = "http://www.timeseriesclassification.com/Downloads/%s.zip" % dataset_name
             for fname in [fname_train, fname_test]:
                 if os.path.exists(os.path.join(full_path, fname)):
                     os.remove(os.path.join(full_path, fname))
             extract_from_zip_url(url, target_dir=self._data_dir, verbose=False)
-        try:
-            data_train = numpy.loadtxt(os.path.join(full_path, fname_train), delimiter=",")
-            data_test = numpy.loadtxt(os.path.join(full_path, fname_test), delimiter=",")
-        except:
-            return None, None, None, None
+
```

Comment:
You changed the behaviour here (returning `None` if the data is not available): have you checked whether doctests should be updated (it seems so, if you have a look at Travis CI failures)?

Reply:
In recent commits, I have reverted the changes back to the try-catch blog not to change the behavior